canopen_402/src/motor.cpp

```diff
@@ -511,6 +511,7 @@ void Motor402::handleInit(LayerStatus &status){
     switchMode(status, No_Mode);
 }
 void Motor402::handleShutdown(LayerStatus &status){
+    switchState(status, no_mode_state_);
```

Comment:
In worst case, this would try to switch on the motor first.
I would move this into  `switchMode`.